### PR TITLE
Create user model

### PR DESCRIPTION
**Pull request by EduardoContreras**
**Summary**
Was added create user model into app and generated the same model to data base with sequelize orm.

**Trello Card**
https://trello.com/c/AOx5qsaA/27-modelo-de-usuario

### CODE REVIEW

**Review by gonziesc — CHANGES_REQUESTED** [review]
Delete your unused dependencies. Also, id call the table name `users`

**Review by gonziesc — APPROVED** [review]
its ok but wait to finish the kickoff changes

**Comment by EduardoContreras** [comment]
@gonziesc waiting for approval kickoff branch for merge.

File: package.json [hunk 1]
```diff
@@ -76,6 +76,8 @@
     "prettier": "^1.8.2",
     "prettier-eslint": "^8.2.1",
     "prompt": "^1.0.0",
-    "sequelize-cli": "^4.0.0"
+    "sequelize-cli": "^4.0.0",
+    "sequelize-test-helpers": "^1.0.3",
+    "sinon": "^6.1.5"
```
Comment (gonziesc) [inline package.json:81]: are you using all of this?

**Review by gonziesc — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (EduardoContreras) [reply package.json:81]: really no, but i will use in the next branch sign-up.

**Pull request merged**